### DISCUSSION TITLE
Moderator for Spanish language.

### DISCUSSION THREAD

**Discussion by juandes** [tensorflow/docs-l10n#449] [General]
Hi,

I was wondering if a moderator is working on the Spanish language. There's a PR I created around a month ago, and I haven't received any feedback.

Thanks.

**Comment by ciliamadani** [comment]
@lamberta Can please help with that ? I need to know the same thing abt the Arabic language, I guess there is no moderator. 
Thank you.

**Comment by lamberta** [comment]
@juandes I see someone from the community has verbally signed off ... are you able to send a pull request from your [GitLocalize review](https://gitlocalize.com/repo/4592/es/review/25104)?

We'll need to figure out a sustainable community review process and work with GitLocalize. Since Spanish is an officially supported language, maybe we can include these in our periodic vendor translations. Arabic is not officially supported and has a very small contributor community (also complications with RTL), so not sure we'll be able to help much there, unfortunately.

**Reply by ilyaspiridonov** [reply]
@lamberta @juandes Won't be able to send a PR as they are not added as a Language Moderator. Would you like me to add them? 
I know we discussed that we'll probably have to let anyone from the community send PRs - so this is coming.

**Reply by ilyaspiridonov** [reply]
On a second thought, we (Alconost) can start helping with high-level reviews of community transltions in 2021.

**Reply by lamberta** [reply]
Great, thanks @ilyaspiridonov. Let's hold off on moderators and take a look at the community workflow in 2021. The majority of contributions come from vendors now, but it would be nice to have the community contribute with their expertise.

**Reply by juandes** [reply]
Hi, @lamberta and @ilyaspiridonov. I want to say thanks for following up on the issue.

**Reply by juandes** [reply]
Hi :) Any updates regarding this? If possible, I'd like to nominate myself for a spot within the Spanish Moderator group. Thanks.

**Reply by lamberta** [reply]
Hi juandes@,
No updates on community moderator status and still not sure it's a route we're going to take. Nonetheless, you should still be able to submit and review using the GitLocalize UI ... and then we'll batch them together when we run vendor jobs.

**Reply by juandes** [reply]
Hi lamberta. Thanks for the quick reply.

I submitted a PR 4 months ago and it's still there. That's what I asked :D. Again, thanks!

**Reply by lamberta** [reply]
Ah, apologies. Still working on the process and what kind of support we'll be able to provide for es in 2021.
